### ISSUE TITLE
Extensions don't load

### ISSUE THREAD

**Issue by svupper** [AbdBarho/stable-diffusion-webui-docker#400]
**Has this issue been opened before?**

- [x] It is not in the [FAQ](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/FAQ), I checked.
- [x] It is not in the [issues](https://github.com/AbdBarho/stable-diffusion-webui-docker/issues?q=), I searched.

**Describe the bug**

extensions don't load after restart, it might a problem in the mounting point

**Which UI**

auto

**Hardware / Software**

- OS:  Ubuntu 22.04
- OS version: Ubuntu 22.04.1 LTS
- Docker Version: 23.0.2
- Docker compose version: v2.17.2
- Repo version: commit sha
- RAM:16
- GPU/VRAM:24

**Steps to Reproduce**

restart the container, extensions error

**Additional context**
![Screenshot from 2023-04-06 19-53-31](https://user-images.githubusercontent.com/56261963/230457589-e8798d8a-2919-4afc-b9b3-e3cb48c50814.png)

**Comment by AbdBarho** [comment]
can you try adding the example `startup.sh` script to your project?

https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Usage#extensions

**Comment by svupper** [comment]
I tried it and it works thanks! I also tried adding 'mkdir data/extensions' to the download/Dockerfile and it's working as well, but I don't really know why...

**Comment by SuperMarioSF** [comment]
I'm facing the same error like this and I guess I found out the reason.

Dynamic Prompt extension doesn't have a `requirements.txt` inside, instead, it has a install.py for requirement installation.

Installing pip packages via `requirements.txt` seems the only way the `/services/AUTOMATIC1111/entrypoint.sh` handles the dependency.

I tried manually writing a `requirements.txt`, listed the package it needed, and it being picked up by `entrypoint.sh` and everything worked.
